### PR TITLE
Add a README file

### PR DESCRIPTION
**Pull request by jack-watts**
Adding README

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+## Legal
+
+**Contribution**
+Before contributing, please ensure that you read the [Contributing](https://github.com/imfug/002-pkg-and-delivery/blob/master/CONTRIBUTING.txt) text that accompanies this repository.
+
+**License**
+[License](https://github.com/imfug/002-pkg-and-delivery/blob/master/LICENSE.txt) terms can be observed [here](https://github.com/imfug/002-pkg-and-delivery/blob/master/LICENSE.txt).
+
+___
+ HPA User Group | imfug.com  | Copyright (c) 2018 Hollywood Professional Association. All rights reserved.
```
Comment (palemieux) [inline README.MD]: This line can be removed since it is already covered in `LICENSE.txt`.

File: README.MD [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+*NOTE: This is not a working document. It is included here for reference purposes until the work of this group is deemed complete.* Please see the [current working version](https://docs.google.com/document/d/1g3E-b52SY98tcREANs1Ud-sQsmRbOM-BMYGpV9UgnIQ).
+
+## Legal
+
+**Contribution**
+Before contributing, please ensure that you read the [Contributing](https://github.com/imfug/002-pkg-and-delivery/blob/master/CONTRIBUTING.txt) text that accompanies this repository.
+
+**License**
+[License](https://github.com/imfug/002-pkg-and-delivery/blob/master/LICENSE.txt) terms can be observed [here](https://github.com/imfug/002-pkg-and-delivery/blob/master/LICENSE.txt).
```
Comment (palemieux) [inline README.MD]: I would not link to `master` and instead simply write:

```
## Legal

Please see LICENSE.txt and CONTRIBUTING.txt before using or contributing to this work.
```

File: README.MD [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+ 1. Create a [new issue](https://github.com/imfug/002-pkg-and-delivery/issues) within this repository clearly stating the
+    issue/contribution as succinct as possible in the title.
+ 2. Elaborate on it in the issue body.
+ 3. Back in the document, comment with the issue ID, linking it with the URL.
+
+*NOTE: Any text comments made in the document itself without a [Github issue](https://github.com/imfug/002-pkg-and-delivery/issues) link risk been overlooked.*
+
+## Communication
+All communication to the group as a whole pertaining to the progress herein will be done so via the designated IMF UG reflector: [supplementals@imfug.com](mailto:supplementals@imfug.com).
+There will be NO communication to the main reflector unless directed to do so by the UG chair. If you are interested in this work, [subscribe](http://imfug.com/mailman/listinfo/supplementals) to the the [mailing list here](http://imfug.com/mailman/listinfo/supplementals).
```
Comment (palemieux) [inline README.MD:19]: We should mention that belonging to the mailing list requires membership to the UG, and provide a link to http://www.imfug.com/join/.

File: README.MD [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+ 1. Create a [new issue](https://github.com/imfug/002-pkg-and-delivery/issues) within this repository clearly stating the
+    issue/contribution as succinct as possible in the title.
+ 2. Elaborate on it in the issue body.
+ 3. Back in the document, comment with the issue ID, linking it with the URL.
+
+*NOTE: Any text comments made in the document itself without a [Github issue](https://github.com/imfug/002-pkg-and-delivery/issues) link risk been overlooked.*
+
+## Communication
+All communication to the group as a whole pertaining to the progress herein will be done so via the designated IMF UG reflector: [supplementals@imfug.com](mailto:supplementals@imfug.com).
```
Comment (palemieux) [inline README.MD]: I would replace `will be` by `is` :)

**Review by palemieux — CHANGES_REQUESTED** [review]

**Comment by jack-watts** [comment]
Minutes have been removed.

**Review by palemieux — APPROVED** [review]

**Pull request merged**